parsing/builtin_attributes.mli

```diff
@@ -40,12 +40,17 @@
 
 
 (** Used by the compiler for attributes that may be compiler built-ins.  These
-    are tracked for misplaced attribute warnings.  This should be used rather
-    than Attr.mk for all built-in attributes. *)
+    are tracked for misplaced attribute warnings (unless this tracking has been
+    turned off with [track_attrs]).  This should be used rather than Attr.mk for
```

Comment:
The last sentence here (not new in this patch) surprises me. If I'm parsing an attribute `f x y [@blah]`, then should I really check whether `blah` is built-in to decide whether to call `Attr.mk` or `mk_internal`?